db/manager.go

```diff
@@ -201,3 +214,42 @@ func getDBConfig(id thread.ID, base *Config, collections ...CollectionConfig) *C
 		Collections: append(base.Collections, collections...),
 	}
 }
+
+// putDBCreds persists db creds for a thread.
+func (m *Manager) putDBCreds(id thread.ID, creds thread.Credentials) error {
```

Comment:
In order to restart DBs, the manager needs to save credentials. This mean storing a public key and the thread signature if they are present.